build/configs/rtl8730e/loadable_ext/defconfig

```diff
@@ -552,6 +552,8 @@ CONFIG_AUDIO_ALC1019=y
 # LCD Driver Support
 #
 CONFIG_LCD=y
+CONFIG_LCD_NORMAL_POWER=90
```

Comment:
Hi @gSahitya-samsung , all the newly added config, you should update the same to Kconfig as well..
Or else, if someone modify the menuconfig on local build, all these parameters will be disappeared in the newly generated .config

Reply:
@edwakuwaku I know, but its not actually decided where to put these config in pm header or defconfig